sdk/workloads/azure-mgmt-workloadssapvirtualinstance/azure/mgmt/workloadssapvirtualinstance/_serialization.py

```diff
@@ -467,7 +467,7 @@ def _classify(cls, response, objects):
 
             if not isinstance(response, ET.Element):
                 rest_api_response_key = cls._get_rest_key_parts(subtype_key)[-1]
-                subtype_value = response.pop(rest_api_response_key, None) or response.pop(subtype_key, None)
+                subtype_value = response.get(rest_api_response_key, None) or response.get(subtype_key, None)
```

Comment:
Before https://github.com/Azure/autorest.python/issues/2428 fixed, we will make the manual patch for now.